### PR TITLE
Styling post index

### PR DESCRIPTION
**Pull request by annalaise**
## Styling of posts - now features collapsible existing comments and new comment field. See images below for full glory 😎 

**Default Collapsed View**
image

**Expanded Comments**
image

**Expanded New Comment Field** 
image

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,42 +1,79 @@
...
+        </span>
+        <br>
+      </div>
+      <div class="post__body card-block">
+        <div class="card-text">
+          <h6><%= post.message %></h6>
+        </div>
+      </div>
+
+      <% unless post.comments.count == 0 && Like.where(post_id: post.id).count == 0 %>
```
Comment (dearshrewdwit) [inline app/views/posts/index.html.erb]: move logic to the model in this if-else statement and test it

**Review by dearshrewdwit — DISMISSED** [review]

**Comment by dearshrewdwit** [comment]
Looks much better!

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (tobold) [reply app/views/posts/index.html.erb]: done!

File: app/models/post.rb [hunk 2]
```diff
@@ -22,6 +22,14 @@ def summary
   end
 
   def user_name
-    "#{user.first_name} #{user.last_name}"
+    "#{user.user_name}"
+  end
+
+  def like
+    if Like.where(post_id: self.id).count == 1
```
Comment (dearshrewdwit) [inline app/models/post.rb]: Can you use rails #pluralize method https://apidock.com/rails/String/pluralize instead

I'm sure you have access to the association method `post#likes` Can you not use this instead?

```ruby
"Like".pluralize(likes.count)
```

Comment (dearshrewdwit) [reply app/models/post.rb]: something like this anyway

Comment (dearshrewdwit) [reply app/models/post.rb]: And make sure both outcomes are tested

**Review by dearshrewdwit — DISMISSED** [review]

File: app/models/post.rb (hunk 2 above)
Comment (tobold) [reply app/models/post.rb]: pluralize method is nifty.

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**